pkg/extensions/search/common/common_test.go

```diff
@@ -52,6 +57,50 @@ type ExpandedRepoInfoResp struct {
 	Errors           []ErrorGQL       `json:"errors"`
 }
 
+type GlobalSearchResultResp struct {
```

Comment:
Why do we need these definitions here instead of using the ones in models_gen.go?

Reply:
In tests, I need a structure to unmarshal the response into. The structures provided by models_gen.go have pointers as fields and the unmarshal library can't work with this kind of structure.